php-practice/section11.php

```diff
@@ -174,15 +187,20 @@ public function bark() {
 $myDog->genus = 'Canine';
 $myDog->species = 'Scary';
 $myDog->barkIsBiggerThanBite = false;
-$myDog->bark(); // What will this print out? ANSWER:
+$myDog->bark(); // What will this print out? ANSWER: ...You better run, man... This is because we have barkIsBiggerThanBite set to False so when function bark() runs the if function runs it sees 
+                /* that $this->barkIsBiggerThanBite is false and goes to the else portion and echo out the string.
 
 /**
  * PROBLEM 4
  *
  * Create a $kangaroo variable as a `new Mammal()` object. Try to set the
- * barkIsBiggerThanBite property on it -- what happens? Why?
+ * barkIsBiggerThanBite property on it -- what happens? Why? When I tried it it said " Call to undefined method Mammal::bark()"
```

Comment:
You are right. I didn't even notice that. I deleted the bark() portion and then tried running the code again with var_dump since it doesn't print anything to the console otherwise. It does allow me to run the barkIsBiggerThanBite.